portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
+
+//Servlet that creates a new comment entity and adds it to the database
+
+@WebServlet("/new-comment")
+public class NewCommentServlet extends HttpServlet {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String author = request.getParameter("cf-name");
```

Comment:
Not sure what 'cf' means in this case.

Reply:
It's the id/name I gave it in the html, "comment-form-name", I'll rename it to that to avoid confusion

Reply:
I had to change these back, it was breaking the entire thing and I can't figure out why. I assume it has something to do with what datastore stores. It was inputting the author and message as blank. I'll work on changing these at another time.

Reply:
How was it breaking?

Reply:
Whenever I would submit a new comment with the new id's it would submit to data store with a blank author an message. I assume it has something t do with the id's being stored somewhere I don't know about, because I changed them everywhere in my code